src/sql/src/plan/statement/show.rs

```diff
@@ -561,8 +561,15 @@ pub fn show_clusters<'a>(
     scx: &'a StatementContext<'a>,
     filter: Option<ShowStatementFilter<Aug>>,
 ) -> Result<ShowSelect<'a>, PlanError> {
-    let query = "SELECT mz_clusters.name FROM mz_catalog.mz_clusters".to_string();
-
+    let query = "
+SELECT
+    mz_clusters.name,
+    pg_catalog.array_agg(ROW(mz_cluster_replicas.name, mz_cluster_replicas.size) ORDER BY mz_cluster_replicas.name)
```

Comment:
Thoughts on making this json instead? That'd allow us to name the fields, easily adding or removing some later on.